mdraid-mirror/boot_drive_howto.rst

```diff
@@ -3,41 +3,46 @@
 Mirroring Rockstor OS using Linux Raid
 ======================================
 
+N.B. this howto is a work in progress and doesn't currently work for a Rockstor
```

Comment:
If we allow ext4 for root filesystem along with btrfs(like it used to be prior to rockstor-core pr 1158, it does work barring the fact that there won't be any root Pool as such. So users won't be able to create Shares or put data on the root filesystem in Rockstor supported ways. But all Pool/Share/Snapshot features work on other drives that may be on the system.

Am I missing something here?